src/Lesson3/Les3_1.java

```diff
@@ -0,0 +1,26 @@
+package Lesson3;
+
+public class Les3_1 {
+
+    public static void main(String[] args) {
+        String word1 = "HELLO";
+        int len1 = word1.length();
+        String word2 = "MY";
+        int len2 = word2.length();
+        String word3 = "FRIENDS";
+        int len3 = word3.length();
+        lengthWords(word1, len1, word2, len2, word3, len3);
+    }
+
+    public static void lengthWords(String word1, int len1, String word2, int len2, String word3, int len3) {
```

Comment:
No needed to add len1, len2, len3 parameters into method. length() method calculates length of words

 public static void findLongestWord(String word1, String word2, String word3) {
        if (word1.length() > word2.length() && word1.length() > word3.length()) {
           System.out.println(word1);
        } else if (word2.length() > word1.length() && word2.length() > word3.length()) {
            System.out.println(word2);
        } else {
            return word3;
        }
    }